fe1-web/src/core/components/QrCodeScanner.tsx

```diff
@@ -50,6 +50,11 @@ const styles = StyleSheet.create({
   },
 });
 
+/**
+ * Attention: react-camera currently has the problem that the onBarCodeScanned property is only set initially, i.e
```

Comment:
I'd suggest adding a date on this comment and a link to the GitHub issue describing this problem